Content.Shared/Lock/Lockpick/LockpickComponent.cs

```diff
@@ -0,0 +1,18 @@
+using Robust.Shared.Audio;
+namespace Content.Shared.Lock.Lockpick;
+
+[RegisterComponent]
+public sealed partial class LockpickComponent : Component
```

Comment:
Use the `<summary>` tags and add a comment to the component itself explaining what it does.

Reply:
i dunno why it doesnt show as like outdated but yeah added the thingi :33